sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
```

Comment:
It looks like the bracket syntax is not unique to pandoc-fignos. Instead, it's a markdown extension called `bracketed_spans` and we should respect them. We'll have other uses, for example setting figure dimensions.

I'd prefer jinja2's [raw block](http://jinja.pocoo.org/docs/2.9/templates/#escaping) for readability than this.